test/replication-luatest/suite.ini

```diff
@@ -2,4 +2,4 @@
 core = luatest
 description = replication luatests
 is_parallel = True
-release_disabled = gh_5295_split_brain_test.lua gh_6036_qsync_order_test.lua gh_6842_qsync_applier_order_test.lua gh_6033_box_promote_demote_test.lua gh_7253_election_long_wal_write_test.lua
+release_disabled = gh_5295_split_brain_test.lua gh_6036_qsync_order_test.lua gh_6842_qsync_applier_order_test.lua gh_6033_box_promote_demote_test.lua gh_7253_election_long_wal_write_test.lua linearizable_read_test.lua
```

Comment:
Nit: Maybe better use this in test code instead:

https://github.com/tarantool/tarantool/blob/7f52f445ed4c3b0f12665de5ae712653591d79b1/test/luatest_helpers/misc.lua#L15

Reply:
I didn't know about that, thanks!
```diff
diff --git a/test/replication-luatest/linearizable_read_test.lua b/test/replication-luatest/linearizable_read_test.lua
index d20420b1d..e6373dd0e 100644
--- a/test/replication-luatest/linearizable_read_test.lua
+++ b/test/replication-luatest/linearizable_read_test.lua
@@ -1,4 +1,5 @@
 local t = require('luatest')
+local misc = require('test.luatest_helpers.misc')
 local cluster = require('test.luatest_helpers.cluster')
 local server = require('test.luatest_helpers.server')
 local proxy = require('test.luatest_helpers.proxy.proxy')
@@ -84,6 +85,7 @@ g.test_wait_others = function(g)
 end
 
 g.test_timeout = function(g)
+    misc.skip_if_not_debug()
     g.servers[1]:exec(function()
         local t = require('luatest')
         box.error.injection.set('ERRINJ_RELAY_FROM_TX_DELAY', true)
diff --git a/test/replication-luatest/suite.ini b/test/replication-luatest/suite.ini
index 2c9ca5557..c2805362f 100644
--- a/test/replication-luatest/suite.ini
+++ b/test/replication-luatest/suite.ini
@@ -2,4 +2,4 @@
 core = luatest
 description = replication luatests
 is_parallel = True
-release_disabled = gh_5295_split_brain_test.lua gh_6036_qsync_order_test.lua gh_6842_qsync_applier_order_test.lua gh_6033_box_promote_demote_test.lua gh_7253_election_long_wal_write_test.lua linearizable_read_test.lua
+release_disabled = gh_5295_split_brain_test.lua gh_6036_qsync_order_test.lua gh_6842_qsync_applier_order_test.lua gh_6033_box_promote_demote_test.lua gh_7253_election_long_wal_write_test.lua
```